### PR TITLE
amber: new package at v16

### PR DESCRIPTION
**Pull request by codeandkey**
This one requires two different tarballs to be smashed together to build, done via resource

### CODE REVIEW

File: var/spack/repos/builtin/packages/amber/package.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+        run_env.set('AMBERHOME', self.prefix)
+        run_env.prepend_path('PYTHONPATH', sp_dir)
+
+    def install(self, spec, prefix):
+        # install AmberTools where it should be
+        install_tree('amber16', '.')
+
+        base_args = [
+            '-noX11',
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/amber/package.py]: Only a single dash for this flag?

File: var/spack/repos/builtin/packages/amber/package.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    resource(
+        name='AmberTools',
+        sha256='7b876afe566e9dd7eb6a5aa952a955649044360f15c1f5d4d91ba7f41f3105fa',
+        url='file://{0}/AmberTools16.tar.bz2'.format(os.getcwd()),
+        destination='.',
+    )
+
+    depends_on('mpi', when='+mpi')
+    depends_on('cuda@7.5.18', when='+cuda')
```
[the remark this answers is not in the document]
Comment (codeandkey) [reply var/spack/repos/builtin/packages/amber/package.py]: Here the cuda variant is inherited from the CudaPackage.

File: var/spack/repos/builtin/packages/amber/package.py (hunk 1 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/amber/package.py]: Yep, the configure script is written by hand and isn't super consistent.

File: var/spack/repos/builtin/packages/amber/package.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+    depends_on('netcdf-fortran')
+    depends_on('python+tkinter', type=('build', 'run'))
```
[the remark this answers is not in the document]
Comment (codeandkey) [reply var/spack/repos/builtin/packages/amber/package.py]: Updated with explicit python version (amber requires 2.7.x)

File: var/spack/repos/builtin/packages/amber/package.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
+    depends_on('py-matplotlib@:2.9', type=('build', 'run'))
```
[the remark this answers is not in the document]
Comment (codeandkey) [reply var/spack/repos/builtin/packages/amber/package.py]: Yes, the package won't seem to concretize with matplotlib >= 3.0 when it tries to bring in python 3.

File: var/spack/repos/builtin/packages/amber/package.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+       Alternatively, add this file to a mirror so that Spack can find it.
+       For instructions on how to set up a mirror, see
+       http://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = "http://ambermd.org/"
+    url      = "file://{0}/Amber16.tar.bz2".format(os.getcwd())
+
+    version('16', sha256='3b7ef281fd3c46282a51b6a6deed9ed174a1f6d468002649d84bfc8a2577ae5d')
+
+    variant('mpi', description='Build MPI executables', default=False)
```
[the remark this answers is not in the document]
Comment (codeandkey) [reply var/spack/repos/builtin/packages/amber/package.py]: Updated!

**Comment by codeandkey** [comment]
Looks like CI timed out, build may need to be retriggered

**Pull request merged**